### PR TITLE
fix(scroll): Update react-hybrid to 0.0.12 

### PR DESCRIPTION
**Pull request by christopherthielen**
v0.0.12 passes `secondary-panel` class through to the `ui-view`

### CODE REVIEW

**Comment by christopherthielen** [comment]
@anotherchrisberry @jrsquared  please check if I missed reverting any CSS you added because of the missing `secondary-panel` class

**Comment by jrsquared** [comment]
LGTM

File: app/scripts/modules/core/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@spinnaker/core",
-  "version": "0.0.35",
+  "version": "0.0.36",
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/package.json]: let's do this in a separate PR for traceability

**Comment by anotherchrisberry** [comment]
LGTM once the core version bump is removed

File: app/scripts/modules/core/.npmignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+yalc.*
+.*
```
Comment (christopherthielen) [inline app/scripts/modules/core/.npmignore]: @anotherchrisberry @jrsquared FYI, this excludes .happypack and .awcache from the npm package
